### DISCUSSION TITLE
Suitable value for sampledFeature in GE CPTSurvey

### DISCUSSION THREAD

**Discussion by HanWelmer** [INSPIRE-MIF/helpdesk#113] [Q&A]
We are mapping an existing dataset of a CPT Survey to INSPIRE Geology. The INSPIRE Datasetspecification Gelogy states:

As a minimum, SF_SamplingFeature requires two properties: shape and sampledFeature. Shape is the geometry of the domain where the sampled values are referenced. SampledFeature can be any feature that is considered as the target of observation. An efficient way of binding geology and geophysics is to refer to one or more geological units. If no specific feature can be named, reference to a generic concept (e.g. http://sweet.jpl.nasa.gov/2.2/realmGeol.owl#Lithosphere ) can be used. ...

<sam:sampledFeature xlink:href=http://sweet.jpl.nasa.gov/2.2/realmGeol.owl#Lithosphere/>

What would be a suitable value for sampledFeature in case of a CPTSurvey, where the shallow subsurface to a depth of, say, 200 meters is investigated on for example bearing capacity?

**Comment by KathiSchleidt** [comment]
Hi Han,
I'm not familiar enough with geology to correctly guide you, but maybe the following will help you to help yourself.
The NASA Sweet Ontology referenced in the INSPIRE specifications has unfortunately been moved, thus none of the URIs we recommended are still active (really wish NASA had the decency to retain these!)
The current location for NASA Sweet is: 
- https://github.com/ESIPFed/sweet

The following viewer allows for nice navigation:
- https://bioportal.bioontology.org/ontologies/SWEET/?p=classes&conceptid=http%3A%2F%2Fsweetontology.net%2FrealmGeol%2FLithosphere

Maybe you can identify a more suitable concept for your work? If so, could you please also provide a link here, to save others the searching?
:)
Kathi
